### ISSUE TITLE
 can only concatenate str (not "NoneType") to str

### ISSUE THREAD

**Issue by bitjerry** [eternnoir/pyTelegramBotAPI#1470]
Please answer these questions before submitting your issue. Thanks!

1. What version of pyTelegramBotAPI are you using?
4.4.0
2. What OS are you using?
win 10
3. What version of python are you using?
3.10

async_telebot.py[line:392] ERROR can only concatenate str (not "NoneType") to str???
I deploy async bot using webhook on server.

When I tested it by myself, the bot worked fine, but when I tested it with other users, they couldn't use it. The bot left this in my server logging. I looked at the bot source code and suspected that there was a problem accepting commands, I I can't reproduce it by myself. My code does not report error, it occurs in async_telebot.py, I really can't find the reason.
Here is my code that accepts the command, maybe it helps to solve the problem.
cmd = ["start", "help"...]
@bot.message_handler(commands=cmd)
async def message_cmd(message):
     users = get_users(message.from_user)
     if users:
         command = message.text[1:]
         uid = str(message.from_user.id)
         if users.cmd.__contains__(command) and check_user(uid):
             asyncio.create_task(users.cmd[command](message), name=uid)

**Comment by coder2020official** [comment]
<img width="516" alt="image" src="https://user-images.githubusercontent.com/63740404/156219711-4368e3e6-17e5-411a-bb95-ccc3f5f81953.png">
Hello, 

Looks like there is an issue with logging line.
Try to print that error you are getting by using:
[Exception handler](https://github.com/eternnoir/pyTelegramBotAPI/blob/master/examples/asynchronous_telebot/exception_handler.py)
Thank you for your report.
I am not sure why Exception is None.

**Comment by Badiboy** [comment]
> Looks like there is an issue with logging line.

No problems can arise from the logging line. It's absolutely correct.

**Comment by coder2020official** [comment]
> > Looks like there is an issue with logging line.
> 
> No problems can arise from the logging line. It's absolutely correct.

Well, I think exception may somehow return None.
Take a look at error:

`async_telebot.py[line:392] ERROR can only concatenate str (not "NoneType") to str`

So, as you see - there is error with concatenation.
And, version 4.4.0, on line 392 has this logging line.

**Comment by bitjerry** [comment]
So sorry, the problem is in my code, not pyTelegramBotAPI.
The reason is that I did not consider that the username of telegram can be empty, username=None
This exception is generated in my processing of the username string
The reason why the error only shows
`async_telebot.py[line:392] ERROR can only concatenate str (not "NoneType") to str`
I think it's because I didn't catch the error in the code that handles the username string, so that the error penetrated the python decorator and was caught until it reached async_telebot. At this time, the error is no longer displayed in my code. , so I mistakenly thought the problem was with pyTelegramBotAPI.
Sorry again!
Below is the error code
```
@bot.message_handler(commands=cmd)
async def message_cmd(message):
    ......
    User.add_user(who.id, '@' + who.username)
```
The real error is in `@+username`
but username may be None

This issue has been resolved, maybe it can be closed
